tests/lib/rules/newline-between-statements.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * @fileoverview Tests for newline-between-statements rule.
+ * @author Toru Nagashima
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/newline-between-statements");
+const RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+ruleTester.run("newline-between-statements", rule, {
```

Comment:
Thank you for the suggestion.
I added many tests from JSCS rules.
I'm continuing to add more tests.